qiita_db/util.py

```diff
@@ -506,13 +506,15 @@ def compute_checksum(path):
     return crc & 0xffffffff
 
 
-def get_files_from_uploads_folders(study_id):
+def get_files_from_uploads_folders(study_id, ignore_hidden_and_folders=True):
```

Comment:
Thought we were moving away from that type of logic for programming, but if everyone else agrees, fine.